client/lib/interpolate-components/Makefile

```diff
@@ -1,7 +1,23 @@
 REPORTER ?= spec
-MOCHA ?= ../../../node_modules/.bin/mocha
+
+ifneq ("$(wildcard node_modules/.bin/)","")
+NODE_BIN ?= ./node_modules/.bin
+else
+NODE_BIN ?= ../../../node_modules/.bin
+endif
+
+BABEL ?= $(NODE_BIN)/babel
+MOCHA ?= $(NODE_BIN)/mocha
 
 test:
 	@NODE_ENV=test NODE_PATH=test:../../../client $(MOCHA) --compilers jsx:babel/register --reporter $(REPORTER)
```

Comment:
resolved in: https://github.com/Automattic/wp-calypso/commit/f8407c1830d2d506f2718d9ff7491cc2a80694f6